src/ui/input/input.tsx

```diff
@@ -0,0 +1,148 @@
+import { ReactNode, ChangeEvent, useState} from "react"
+import { Icon } from "../icon/Icon"
+import style from "./input.module.scss"
+import cnBind from 'classnames/bind'
+
+const cx = cnBind.bind(style)
+
+interface InputProps {
+    className?: string
+    placeholder?: string
+    type?: 'text' | 'password' | 'textarea' | 'number'
```

Comment:
```suggestion
    type?: 'text' | 'password' | 'textarea' | 'number' | 'email'
```
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types

Reply:
исправлено